### DISCUSSION TITLE
Getting  a TypeError draw_callback_3d (missing self and context) in console

### DISCUSSION THREAD

**Discussion by slopcop** [carson-katri/dream-textures#525] [Q&A]
So, I get this error: 
`TypeError: draw_callback_px() missing 2 required positional arguments: 'self' and 'context'`

When I click to download a model in addon prefs (key from hugging space there - tried read and write), it changes the interface for a split second then goes away and all I see is a flood of the above error.

Now, I'm using the 3.5 alpha, so that might be it.  Apologies if I missed that 3.4 is required.  Importing checkpoints hasn't worked so far (stable 1.5, 1.4, and as many others as I could find :) I have 20+ models - mostly the major ones)

**Comment by carson-katri** [comment]
This is one possible solution: https://github.com/carson-katri/dream-textures/issues/516#issuecomment-1383152411
